### DISCUSSION TITLE
How do you create a dropdown without a button? Or a modal that can attach relatively, like the select window?

### DISCUSSION THREAD

**Discussion by matthew-dean** [epicmaxco/vuestic-ui#3144] [Q&A]
This is driving me bonkers, because this seems possible in every other UI library, but I can't figure out how to do it.

I want to:
1. Create a dropdown that I can attach to any arbitrary element and trigger programmatically.
2. I do not want to render a button.
3. I want to be able to make it full-screen (unattached) if I need to.

Usually there's a "menu" component that can just be attached to things. But it seems like all of the dropdowns in Vuestic are hard-coded to the rendering of one particular element. Is this the case? If so, why is that the case?

**Comment by m0ksem** [comment]
You can use https://vuestic.dev/en/ui-elements/dropdown to make a custom dropdown.
